GAE/war/WEB-INF/webapp-security.xml

```diff
@@ -42,6 +42,9 @@
         <intercept-url pattern="/rest/users" access="hasRole('USER')"/>
         <intercept-url pattern="/rest/user_roles" access="hasRole('ADMIN')"/>
         <intercept-url pattern="/rest/user_auth" access="hasRole('ADMIN')"/>
+        <intercept-url pattern="/rest/reports" method="GET" access="hasRole('USER')"/>
+        <intercept-url pattern="/rest/reports/*" method="PUT" access="permitAll"/>
+        <intercept-url pattern="/rest/reports" method="POST" access="permitAll"/>
```

Comment:
I believe we should simply have a generic rule that one has to have a `USER` role to be able to execute any of the method on this endpoint.

Reply:
That was to let the engine do a PUT, since it is not Spring authorized.

Reply:
Understood.  We will need to have a rule `/rest/reports` rule for the UI to authorized through spring.